### DISCUSSION TITLE
API call to obtain table with hourly forecast only

### DISCUSSION THREAD

**Discussion by franktoffel** [open-meteo/open-meteo#1052] [Q&A] [answered]
[API doc ](https://open-meteo.com/en/docs) shows how to get the API call and a preview of the trends you will get.

However, when using the URL, a JSON with three elements appear. 

- location (latitude, longitude, elevation....)
- units of for the hourly data
- hourly data (time, temperature, humidity...)

Is there an option to modify the API call so only only the hourly data table is obtained? 

It will be even better to get location as additional columns that don't change over time in this second table.

**Comment by patrick-zippenfenig** [comment]
Do you mean CSV? JSON has no tables

**Reply by franktoffel** [reply]
Yes. Having the hourly table (and others) as a csv will perhaps help. It will simplify the data steps that needed to parse JSON. You already have this option to download csv or excel already, not sure if this can be done from the API directly.

**Reply by patrick-zippenfenig** [reply, marked as answer]
If you add `&format=csv` to the API URL, it will return data in CSV. It will directly prompt you to download a file, because the HTTP content disposition headers are set. E.g. https://api.open-meteo.com/v1/forecast?latitude=52.52&longitude=13.41&hourly=temperature_2m&format=csv

 Otherwise as @guidocioni wrote, use Python to format the data however you like to :)

**Comment by guidocioni** [comment]
You can easily parse the `hourly` path of the JSON response with Python and ignore the rest.
